helpers/postgresql_dump.sh

```diff
@@ -1,12 +1,34 @@
 #!/bin/bash
+# Script to create backup files for rhel6 to 7 migration.
+#
 # Notes:
-# 1. Use this script to create postgresql dumps - foreman.dump and candlepin.dump from the Satellite server.
-# 2. This scripts stops the katello-service in Satellite.  Please plan accordingly for the outage.
+# 1. This script runs katello-backup and also creates foreman.dump and candlepin.dump files from your Satellite6 server.
+# 2. Run this script as root user in Satellite6 server.
+# 3. This script stops the katello-service in Satellite.  Please plan accordingly for the outage.
+
+set -x
+
+# Create backup folder
 export backup_dir="/backup"
 mkdir $backup_dir
+
+# Run katello-backup
 katello-backup $backup_dir
+echo "pgsql_data.tar.gz is not required for rhel6 to 7 migration..."
+rm -f $backup_dir/katello-backup*/pgsql_data.tar.gz
```

Comment:
@sthirugn if you want to include it, it looks like this will work
```
[root@sat-628-rhel7 /]# backup_dir=$(cat /tmp/backup_text.txt | awk '{w=$9} END{print w}')
[root@sat-628-rhel7 /]# cd $backup_dir
[root@sat-628-rhel7 katello-backup-2017-03-29T19:20:12+00:00]# 
```
That is just from the saved output from the file so you will have to replace that cat part and probably change the variable name since its  already used by you